packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Basic.Example.tsx

```diff
@@ -74,7 +74,7 @@ export class CalloutBasicExample extends React.Component<{}, ICalloutBasicExampl
     return (
       <>
         <div className={styles.buttonArea} ref={this._menuButtonElement}>
-          <DefaultButton onClick={this._onShowMenuClicked} text={isCalloutVisible ? 'Hide callout' : 'Show callout'} />
+          <DefaultButton onClick={this._onShowMenuClicked} text={isCalloutVisible ? 'Hide Callout' : 'Show Callout'} />
```

Comment:
This should stay sentence-case rather than title case. MS style is to use sentence case, even though we're not consistent about that right now.

Reply:
I capped it because "Callout" is a proper noun.

Reply:
In some sense that's true, but the style we're moving towards (per editorial board discussions) is using sentence case names rather than CodeNames for components unless it's necessary for clarity. Though that's still very inconsistent in the current docs.